### DISCUSSION TITLE
Ausstehende Geräteupdates werden nicht mehr auf der Startseite aufgeführt

### DISCUSSION THREAD

**Discussion by Weseler485** [OpenCCU/OpenCCU#2544] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

Seit dem Update auf die neuste Version wird auf der Startseite nicht mehr angezeigt, dass für srei Geräte eine aktuellere Firmware auf der Zentrale verfügbar ist. In der Geräteübersicht ist dies weiterhin ersichtlich.

### Describe the behavior you expected

Weiterhin der Hinweis auf der Startseite, wenn auf der CCU eine Firmwareversion geladen ist, welche noch nicht auf jedem Device geladen wurde.

### Steps to reproduce the issue

1. Aktuellere Devicefirmware auf CCU hinterlegen
2. 3.73.9.20231130 installieren

### What is the version this bug report is based on?

3.73.9.20231130

### Which base platform are you running?

rpi3 (RaspberryPi3)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
.
```

**Comment by jens-maus** [comment]
Bitte konkrete Screenshosts davon zeigen und auch sicherstellen das der Browser der auf die WebUI gerade zugreift auch entsprechend ins Internet gelangt. Auch bitte ggf. die Browserkonsole nach Fehlern kontrollieren. Hier funktioniert das ganze tadellos.

**Comment by Weseler485** [comment]
![image](https://github.com/jens-maus/RaspberryMatic/assets/79879510/451f3bef-c6c7-4426-9477-d8246b464394)
![image](https://github.com/jens-maus/RaspberryMatic/assets/79879510/5761dc5c-e33a-486b-8831-63aae2c4c5bb)

![image](https://github.com/jens-maus/RaspberryMatic/assets/79879510/fe3ba335-fe7d-48a8-92a4-c8b7d1116bb9)
![image](https://github.com/jens-maus/RaspberryMatic/assets/79879510/bfbc0c0e-2bd9-4a9d-b31c-82da4d4fca1b)

Noch etwas?

**Comment by jens-maus** [comment, marked as answer]
Nun, dann blockt wohl irgendetwas (deine Firewall, dein Browser, etc.) die URL zum abfragen der Device Firmwares. Geb einfach mal die URL in einen anderen Browser/anderes System ein und du wirst sehen das dann das JSON mit den informationen über die aktuelle  Gerätefirmwares zurückgeliefert werden. Hier klappt das jedenfalls problemlos. Sieht mir als nach einem rein lokalen Problem auf deiner Seite aus.

**Reply by Weseler485** [reply]
Dann mag es ggf. am VPN, der Firewall oder was auch immer liegen. Ist jedenfalls seit dem Update neu.
Wenn ich den Port in der URL entferne, dann funktioniert es und ich bekomme eine JSON geliefert.
Ist der Port wichtig (HTTPS scheint ja auch ohne zu funktionieren) oder kann man den entfernen?

**Reply by jens-maus** [reply]
Na wenn das nur mit port 443 statt 8443 geht riecht das arg danach das irgendwas auf deiner seite blockiert das du via port 8443 zugreifen kannst auf `ccu3-update.homematic.com`. Oder du greifst einfach zu oft darauf zu und es schlägt ein rate limiting auf seiten des eQ3 servers zu... Nen Bezug auf die aktuelle Version bzgl. "Ist jedenfalls seit dem Update neu", gibt es nicht. Dahingehend wurde nichts geändert. Zufall oder eben auf deiner Seite was verändert.

Und das man Port 8443 nutzen soll kommt eben von eQ3 bzw. ist so in der CCU3 Firmware bzw. OCCU auch genutzt. Kann natürlich sein das nun auch der pure 443 Port geht und das wäre in der Tat natürlich besser. Aber dazu habe ich noch keine Info das dieser stattdessen nun genutzt werden sollte...

**Reply by Weseler485** [reply]
Naja, dass das so ganz gar nichts mit dem Update zu tun hat, bezweifele ich. Zumindest im Commit 6d7d4741f3fff8d83edf6a2593cd19f7f50a44a3 ist da was am Patch 0181 geschraubt worden, dass DeviceFirmwares über HTTPS abgefragt werden. Vermutlich ist es bei mir bisher immer über reines HTTP Port 80 gelaufen.
Mag zwar unterm Strich noch immer ein UserProblem sein, aber erst und "verursacht" durch das neue Release :)

**Reply by jens-maus** [reply]
Stimmt. Vermutlich wird es in der Tat daran liegen das du bzw. irgendeine Maßnahme in deinem Netzwerk oder in deinem System es vielleicht verhindert das Port 8443 verwendet wird und nun fällt das erst wirklich auf weil eben nun immer https mit port 8443 für das abfragen der Gerätefirmwares verwendet wird. Müsste ich dann in der Tat mal nachschauen ob ich nicht einfach den Standard 443 Port nehmen kann wenn dieser jetzt für diesen eQ3 update Host anscheinend das gleiche Ergebnis liefert...

**Comment by UweHalla** [comment]
Hallo. Ich beziehe mich mal auf diesen Thread da es bei mir genau so aussieht. Entstanden ist das nachdem ich gezwungen war ein Backup zurückzuspielen. Ich hatte letzte Woche noch gesehen das einige Updates angezeigt wurden, nun nach dem Restore sind diese nicht mehr sichtbar. Ich habe also an meiner Infrastruktur keinerlei Änderungen vorgenommen.

**Reply by jens-maus** [reply]
Auch einfach mal abwarten. Die Anzeige ob es neue Geräte-Firmware-Updates gibt oder nicht wird nur in gewissen Intervallen aktualisiert und nicht jedesmal wenn du dich in die WebUI einloggst. Also einfach mal nen halben Tag warten nach einem frischen Neustart der Zentrale und gut ist. Auch hier ist Geduld eine Tugend ;-)

**Comment by tb4jc** [comment]
Hallo zusammen
Ich habe noch einen Zusatz.
Bei mir hat der Geräteupdate auch lange nicht mehr funktioniert. Ich bekam keine Meldung mehr über neu verfügbare Firmware Updates.
Mit Hilfe obiger Bilder der Dev Console und dem Code webui.js habe ich nun herausgefunden, dass die falsche URL "fieldtest-ccu3-update.homematic.com" benutzt wurde, weil die Datei /usr/local/etc/config/fieldTestActive existierte.
Keine Ahnung woher diese Datei auf einmal kam, ich muss wohl irgendwie auf folgende Einstellungen gekommen sein.
![image](https://github.com/user-attachments/assets/09765dd3-95ee-44d2-9b82-ace3b1c9e5b0)
Vielleicht hilft jemandem diese Meldung.

**Reply by Joell86** [reply]
Danke! Genau dieses Feld war bei mir auch gesetzt. Direkt nach dem Ausschalten FW-Updates für 23 Geräte bekommen :D
